sites/partners/src/listings/PaperListingForm/sections/LeasingAgent.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react"
+import { useFormContext } from "react-hook-form"
+import { t, GridSection, Textarea, Field, PhoneField } from "@bloom-housing/ui-components"
+
+const LeasingAgent = () => {
+  const formMethods = useFormContext()
+
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, control } = formMethods
+
+  return (
+    <div>
+      <GridSection grid={false} separator>
```

Comment:
We could do something similar here to my comment on the detail pages for the detail forms too.